### ISSUE TITLE
SW특기자전형에 대해서

### ISSUE THREAD

**Issue by sunrinio** [sunrinio-org/QnA#72]
카이스트나 고려대같은 곳을 특전으로 갈 수 있다고 하는데 현실성이 어느정도 되나요?

**Comment by wookje** [comment]
더도 말고 덜도 말고 자기가 하는 만큼 갑니다. 특별전형이라고 요행은 없어요.

**Comment by nerogit** [comment]
@wookje 님 말 ㄹㅇ동감합니다. 그쪽 입학팀도 보는 눈이 있을텐데 특전이라도 갈만한 사람이 갈듯요ㅋㅋ

그리고 그정도 노려볼정도로 잘하면 아마 워낙 다른길이 많아서 행복한 고민속에서 살지 않을까 싶습니다

**Comment by nerogit** [comment]
아 또 그정도 실적 내려면 영어&수학공부도 안할수가 없을거에여.. 혹시 공부안하고 컴터만해서 대학잘갈수있냐는 질문이었으면 전 단호하게 ㄴㄴ라고 할검니다

**Comment by HealGaren** [comment]
저는 좀 의견이 다른게
SW특기자전형은 이번 고3 되는 학년부터 처음 시작하는 전형입니다..ㅜㅜ 요강도 아직 확실하게 내놓지 않고 있고요..
경험해보신 분들이 있을 수가 없고 카드는 까보기 전에는 모를 것 같아요.

**Comment by nerogit** [comment]
@HealGaren 님 의견도 일리가 있는데 저는 아무리 입시요강이 바뀌고 전형이름이 바뀌어도 대학에서 원하는 학생상은 크게 바뀌지 않을 것 같아요. 전 입시준비할때 **대학에선 어떤 학생을 원할까** 라는 생각을 정말 많이 해봤었는데 ~공부하는시간보다 저 생각하는 시간이 더 많았음여~ 그 고민이 꽤 의미있었던것 같네요.

**Comment by nerogit** [comment]
@therne 님도 생각 많이 해보셨을거같은데 한마디 해주시죠

**Comment by MuhunKim** [comment]
@nerogit 님 말에 덧붙이자면 학교마다 원하는 인재상이 달라서 캐바캐(`case-by-case`)이죠.

**Comment by nerogit** [comment]
맞아요... 사실 케바케라 저도 저런 고민이 결과에 영향을 미쳤다기보단 자소서 쓸때 어떤 방향으로 써야할지 뭐 그런것들때문에 의미있었어요.

사실 위험해요ㅠㅠ 그런 전형 한두개만 보고 고3 생활한다는게.. 뭐 아무쪼록 파이팅 하셨으면 좋겠네요
~저는 스무살되는 1월 1일에 다신 입시계랑 엮이지 말아야지 라는 굳은 다짐을 했읍니다~

**Comment by jen6** [comment]
확실한건 공부해서 카이스트 고대 가는것보다 어려우면 어려웠지 쉽지는 않을겁니다.
다만 서울 중위권 대학들이라면 좀 얘기가 달라질수도 있다고 생각해요. 국민대 이민석 교수님이 저번에 달았다는 댓글기준으로는 정말 코딩만으로 뽑으신다는 얘기를 하시기도 하셨으니까요.

**Comment by nerogit** [comment]
~그럼 내가 뽑혔을리가 없는데~

**Comment by therne** [comment]
아쉬운 현실이지만... 내신이 안되신다면 특기자 기대하진 마세요... ㅜㅜ

한 친구는 ISEF 국대출신에 이미지 프로세싱 연구해서 정올에 학회에서 상도 받은 친구이고 저도 딥러닝과 알고리즘 관련해서 논문 제출하고 컴파일러에 뭐에 이것저것 제출해봤지만 결국에 뽑히는 놈은 **내신도 스펙도 좋은놈**이뽑혀요.. 저는 4군데 광탈했고 친구는 7군데 광탈했습니다.

16년도 입시 데이터를 볼때 각 대학들 특기자전형 최소요구조건을 보자면,
* 카이스트 특기자 - 내신 1점대가 최소요구조건
* 연세대 창의IT - 특성화 안뽑음
* 고려대 과학인재 - 특성화 안뽑음 (혹은 1점대 초반?)
* 유니스트 특기자 - 내신 2등급이 최소요구조건

이정도 맞추신다음에 스펙을 생각하셔야 할 것 같아요. 이 밑으로는 아무리 스펙이 좋더라도 떨어졌습니다.

올해 신설되는 한양대나 성대는 잘 모르겠지만 그 대학들의 기존 수시 기조를 봤을때 한양대는 모의고사 성적까지 보던 학교였고, 성대도 성적면에선 친절한 학교는 아니라 낙관적으로 볼 순 없을것 같습니다.

좋은 대학을 가시려면 그 대학이 원하는 조건을 맞춰야 합니다. 대학이 맞춰주는 경우는 별로 없어요...

**Comment by MuhunKim** [comment]
@jen6 후자 중에 @nerogit 님이 그 뽑힌 사람이죠 ㅎㅎ

**Comment by codertimo** [comment]
이번 특기자전형 (카이 유니 디지 지스 연세 고대 서강(3개) 국민) 10개 다 넣어서 9개 광탈한 닝겐입니다..

내신이 3점 후반대였고, 영상처리쪽을 연구했고 국제대회 CS분야에 한국대표로 출전하기도 했었습니다.

이 주제에 대해서 한마디로 요약하자면 이번년도는 내신이었습니다. 컴퓨터 분야를 깊게 판 학생이 아니라, 내신이 좋른 학생중 컴퓨터분야는 조금 백그라운드(수상실적,포폴)이 있는 학생들을 뽑았던것 같습니다

물론 이건 2016년의 실험결과입니다. 2017년도에 신설되는 한양대 성대는 어떻게 될지 모르겠습니다.

하지만 그 이전에 제가 언급했던 9개의 학과는 내신이 베이스가 되어야 한다는 것을 뼈저리게 느꼈습니다.

지원하고자 하는 학생은 참고하시길 바랍니다..
(국민대는 컴터쪽 잘하는 학생이면 내신 거의 밑바닥 아니면 뽑습니다 ㅎ)

**Comment by imkimchi** [comment]
미국으로 오세요
